golang/Makefile

```diff
@@ -53,7 +53,7 @@ upd:
 .PHONY: down
 down:
```

Comment:
We talked about this, up and down should be in pairs.

Reply:
Done.